website/pages/gamejam_tools_en.md

```diff
@@ -23,22 +23,35 @@ These tools works great for game jams and are mostly free.
 - [Krita (2D drawing and animation)](https://krita.org/)
 - [MakeHuman (3D-models of humans)](http://www.makehumancommunity.org/)
 - [Simplygon (3D-model optimization)](https://www.simplygon.com/)
+- [Mixamo (Autorig and animate humanoid 3D characters)](https://www.mixamo.com/)
 
 ### Tools for making sound effects
 
 - [bfxr (Online old school sound effect generator)](http://www.bfxr.net/)
 - [Sunvox (Modular synthesizer / tracker for music)](http://www.warmplace.ru/soft/sunvox/)
 - [BeepBox (Online chiptune music tracker)](http://www.beepbox.co/)
+- [ChipTones (Old school sound effect generator)](https://sfbgames.itch.io/chiptone)
 
 ### Free game resources
 
 - [OpenGameArt.org (Public domain and creative commons game assets)](http://opengameart.org/)
-- [Itch.io Game Assets](https://itch.io/game-assets)
+- [Itch.io Game Assets](https://itch.io/game-assets/free)
+- [HDRI Haven (HDR / Environment Maps)](https://hdrihaven.com/)
+- [cgbookcase.com (Textures)](https://www.cgbookcase.com/textures)
+- [Texture Haven (Textures)](https://texturehaven.com/)
+- [CC0 Textures (Textures)](https://cc0textures.com/)
+- [textures.com (Textures)](https://www.textures.com/)
+- [Dev Assets](https://devassets.com/)
+- [game-icons.net (Icons)](https://game-icons.net/)
+- [Gameplay Design Patterns](http://virt10.itu.chalmers.se/index.php/Main_Page)
 
 ### Tools for keeping track of work
 
-- [git](https://git-scm.com/)
+- Hosts
 	- [GitLab](http://gitlab.com/)
 	- [GitHub](http://github.com/)
-	- [LiU GitLab](https://gitlab.liu.se/)
+	- [Linköping University GitLab](https://gitlab.liu.se/)
+
+- Clients
+    - [Git](https://git-scm.com/)
```

Comment:
Use tab for indentation to match other lines! (Can't achieve that myself in github's web view)